templates/solr.yaml

```diff
@@ -1,12 +1,33 @@
 AWSTemplateFormatVersion: '2010-09-09'
-Description: Hyku SolrCloud stack using ECS
+Description: Hyku solrcloud stack
```

Comment:
Perhaps "Hyku Solr instance", given that there is now a distinct SolrCloud template?